maven/lib/dependabot/maven/metadata_finder.rb

```diff
@@ -104,7 +105,7 @@ def source_from_anywhere_in_pom(pom)
       def dependency_pom_file
         return @dependency_pom_file unless @dependency_pom_file.nil?
 
-        response = RegistryClient.get(
+        response = Dependabot::RegistryClient.get(
```

Comment:
This code could stay unchanged right? As we're already in the `Dependabot` module. I don't mind if you prefer this for clarity though

Reply:
Yeah, I could have left this but there is a Terraform::RegistryClient which will likely need to extend this class, so I err'd on the side of the full namespace in case we ever insert other extensions

Reply:
Aye, that makes sense 👍